src/test/java/bio/terra/app/configuration/OpenIDConnectConfigurationTest.java

```diff
@@ -29,4 +30,27 @@ public void testInitializeWithGoogleOidc() {
         openIDConnectConfiguration.getTokenEndpoint(),
         equalTo("https://oauth2.googleapis.com/token"));
   }
+
+  @Test
+  void testGetOIDCMetadata() {
```

Comment:
Could you add a test for the case where there is not profile (since that's the production usecase)